### DISCUSSION TITLE
CORS issue with Stripe

### DISCUSSION THREAD

**Discussion by NetOpWibby** [sveltejs/kit#3031] [Q&A]
Does anyone have advice on how to best integrate Stripe with SvelteKit? I'm getting CORS issues.

> Blocked a frame with origin "`https://js.stripe.com/`" from accessing a cross-origin frame.

(Also get this message with `https://m.stripe.network`)

**Comment by antony** [comment]
This isn't a SvelteKit issue per-se, it's a cross-origin issue. There's plenty on stack overflow about it, and you can read more here https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

**Reply by NetOpWibby** [reply]
I know what CORS is, my question is how do I fix it? I've seen people talk about adding a proxy to a SvelteKit app but no examples.

**Reply by antony** [reply]
This part of the documentation might be useful to you https://kit.svelte.dev/docs#hooks-externalfetch

**Reply by NetOpWibby** [reply]
`externalFetch` seems to assume integration with your own API.

**Comment by nhe23** [comment]
Have you seen this repo: [https://github.com/srmullen/sveltekit-stripe](https://github.com/srmullen/sveltekit-stripe)? I tried it out myself and it worked pretty well

**Reply by NetOpWibby** [reply]
I tried integrating into my own project but then decided to just run that repo vanilla. Same result.

<img width="1646" alt="SS 2021-12-17 at 11 44 49 AM" src="https://user-images.githubusercontent.com/1288356/146599478-96833f68-0491-4c5d-9039-69a642694c43.png">

**Comment by CaptainCodeman** [comment]
Normally you just include a normal script tag in the HTML page, e.g. `<script src="https://js.stripe.com/v3/"></script>`

If you are trying to load it via fetch or import it from your own script, that isn't going to work.

They do also provide a separate module if you want to incorporate it into your own app script instead of loading it separately.

See: https://stripe.com/docs/js/including

**Reply by NetOpWibby** [reply]
Not sure where the disconnect is on my system then. I've even cloned [another project](https://github.com/sveltejs/kit/discussions/3031#discussioncomment-1838321) but am still seeing the same issue.

**Comment by antony** [comment]
I will look at that project when I get a minute and see if I get the issue.

I load stripe in this way for Beyonk and it works without issue.
